fastapi_endpoint.py

```
@@ -0,0 +1,127 @@
+# app.py
+import os
+import shutil
+import logging
+import time
```

Ruff (0.8.2)

`time` imported but unused

Remove unused import: `time`

(F401)